bottlecap/src/config/mod.rs

```diff
@@ -188,6 +188,7 @@ pub fn get_config(config_directory: &Path) -> Result<Config, ConfigError> {
 }
 
 #[allow(clippy::module_name_repetitions)]
+#[derive(Debug, Clone)]
```

Comment:
Do we need to add clone trait here?
I thought the configs were immutable and passed as ref/Arc everywhere